library/CHART-TEMPLATE/templates/example-secret.yaml

```diff
@@ -0,0 +1 @@
+{{- include "common.licenseKey.secret" . }}
```

Comment:
I have mixed feelings about this, I see the point of unifying the secret management, but I do not like especially to have a whole yaml in the helpers.

Have you checked if there is any different among the license secrets of the charts? (I believe there is not)

Reply:
> I do not like especially to have a whole yaml in the helpers

I also think this is a bit weird, but I think there is no other way to abstract the chart writers from the logic of whether or not create the secret, and which name it should have. This is a bit "darker" but also leaves very little room for mistake to chart writers.

> Have you checked if there is any different among the license secrets of the charts? (I believe there is not)

I think we have a couple of charts that use APIKeys and such. For those, we can either incorporate that logic to this secret, or just use other secret for that.

Reply:
I see the point of having it and we can mark is as solved, I was just a bit worried that is quite different from all templates we own. 

Maybe @kang-makes has some ideas about this.

Reply:
Oh no, you summoned the nitpicking beast... My pr is doomed /s